### PR TITLE
Add multi-data center commands

### PR DESCRIPTION
**Pull request by tzumainn**
* view multiple offers
* claim multiple offers
* view multiple leases

### CODE REVIEW

File: esileapclient/osc/v1/mdc/mdc_lease.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+        parser.add_argument(
+            '--clouds',
+            dest='clouds',
+            metavar='<clouds>',
+            nargs="+",
+            help="Specify the cloud to use from clouds.yaml."
+        )
+        parser.add_argument(
+            '--long',
```
Comment (ljmcgann) [inline esileapclient/osc/v1/mdc/mdc_lease.py:42]: Are we ok with displaying detailed information, when the fields of a lessee could update what the columns are? Two datacenters could be running different versions of esileap. We purposely didn't do this for ironic after all, though Ironic does change faster than esileap, and also we are the ones responsible for changing esileap

File: esileapclient/osc/v1/mdc/mdc_offer.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            parsed_args.end_time else None,
+            'resource_type': parsed_args.resource_type,
+        }
+
+        available_offers = []
+        for c in cloud_regions:
+            client = esileapclient.Client(session=c.get_session())
+            offers = client.offer.list(filters)
+            for offer in offers:
+                offer.cloud_region = c
```
Comment (ljmcgann) [inline esileapclient/osc/v1/mdc/mdc_offer.py:184]: why is this being set? I see we use it to get a session, but why do we need individual sessions for every offer if multiple of the offers could all be within one of the datacenters

**Review by ljmcgann — COMMENTED** [review]
looks good, left two comments

File: esileapclient/osc/v1/mdc/mdc_lease.py (hunk 1 above)
Comment (tzumainn) [reply esileapclient/osc/v1/mdc/mdc_lease.py]: Good point! I've removed the '--long' option.

File: esileapclient/osc/v1/mdc/mdc_offer.py (hunk 2 above)
Comment (tzumainn) [reply esileapclient/osc/v1/mdc/mdc_offer.py:173]: Right now we just grab X random offers, which come out in any order. I agree that in the long run there's probably a better algorithm, and people will probably prefer nodes to be constrained to one data center. But I'd like to wait for outside requirements before doing anything more complicated.

**Comment by tzumainn** [comment]
Thanks for the review! Addressed the comments.

**Review by ljmcgann — APPROVED** [review]

**Comment by mnbatra** [comment]
everything looks good!

**Review by mnbatra — COMMENTED** [review]
everything looks good!

**Pull request merged**